infrastructure/batch.tf

```diff
@@ -26,8 +26,8 @@ module "batch_job_definition" {
   fargate_flag            = false
   env_vars                = local.batch_env_vars
   additional_iam_policy   = aws_iam_policy.batch.arn
-  task_memory_requirements = local.memory
-  task_vcpu_requirements   = local.vcpus
+  task_memory_requirements = local.batch_memory
```

Comment:
Assume you meant to add the infra changes in the same PR?

Reply:
Yes, @duncanjbrown has pulled infra changes to this PR